data-prepper-plugins/service-map-stateful/src/main/java/com/amazon/dataprepper/plugins/prepper/ServiceMapStatefulPrepper.java

```diff
@@ -120,6 +135,7 @@ private static File createPath(File path) {
                                         if (span.getParentSpanId().isEmpty()) {
                                             try {
                                                 currentTraceGroupWindow.put(span.getTraceId().toByteArray(), span.getName());
+                                                uniqueTraceGroupsCounter.increment();
```

Comment:
This is not unique trace groups. This number will be equal to the unique number of traces